Tasks/AzureCLIV1/task.json

```diff
@@ -2,7 +2,7 @@
     "id": "46E4BE58-730B-4389-8A2F-EA10B3E5E815",
     "name": "AzureCLI",
     "friendlyName": "Azure CLI",
-    "description": "Run Azure CLI commands against an Azure subscription in a Shell script when runnning on Linux agent or Batch script when running on Windows agent.",
+    "description": "Run Azure CLI commands against an Azure subscription in a Shell script when running on Linux agent or Batch script when running on Windows agent.",
```

Comment:
Thanks for fixing this. Can you also modify it [here](https://github.com/Microsoft/azure-pipelines-tasks/blob/3c4597b458d588385469ad748c96ca6db972d0b5/Tasks/AzureCLIV1/Strings/resources.resjson/en-US/resources.resjson#L4)? Or you can run "node make.js build --task AzureCLIV1" which will auto-generate it.

Reply:
Ping!